modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc

```diff
@@ -81,19 +105,36 @@ projects:
       type: git
       location: 'https://github.com/eclipse-che/che-server.git'
 ----
-See the xref:authoring-devfiles-version-1.adoc[Devfile reference].
+For more information about devfiles, see xref:authoring-devfiles-version-2.adoc[Authoring a devfile 2.1.0].
 ====
 
+. Click btn:[Save].
+
 [id="editing-the-commands-after-importing-a-project_{context}"]
-=== Editing the commands after importing a project
+=== Adding commands to an imported project
+//Is this section only relevant as a subsection here or does it also apply to the other sections in this file? If the latter, consider promoting it to the next heading level. In any case, not checked by me, max-cx
+
+You can add commands to your project to run, debug, or start your application in a browser.
+
+.Prerequisites
+
+* A workspace with a project in it
```

Comment:
.

Reply:
Period omitted deliberately. IBM Style Guide: p. 89 "_Write lists so that all or none of the items start with complete sentences. ... If list items comprise only sentence fragments, do not include any end punctuation_."